gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -834,7 +834,7 @@ def main(args):
     test_gene = args.test_gene
     overwrite = args.overwrite
     apply_release_filters = not args.skip_apply_release_filters
-    pops = list(set(POPS["v3"] + POPS["v4"]))
+    pops = list(set(POPS["v3"] + POPS["v4"]["exomes"]))
```

Comment:
This will need to change based on my comment in gnomad_methods but then it looks good!